terraform/modules/benchmark/service/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+AllCops:
```

Comment:
Shouldn't we have a shared `.rubocop.yml` in the repo root?

Reply:
yes that will be in another PR[ Enable rubocop for entire forest-iac repo](https://github.com/ChainSafe/forest-iac/pull/229 )